src/OpenLoco/src/Objects/ObjectManager.cpp

```diff
@@ -936,8 +938,45 @@ namespace OpenLoco::ObjectManager
     // 0x004796A9
     void updateYearly1()
```

Comment:
Can we rename this to `updateLevelCrossingDesign`?

Reply:
`defaultLevelCrossing`?

Reply:
```suggestion
    void updateDefaultLevelCrossingType()
```